src/app/routes/connector-ui/policy-definition-page/new-policy-dialog/new-policy-dialog.component.html

```diff
@@ -1,13 +1,13 @@
 <h1 mat-dialog-title>Create New Policy</h1>
 <mat-dialog-content class="w-[800px]">
-  <div class="flex flex-col mt-[10px]">
+  <div class="flex flex-col">
     <!--  ID -->
     <mat-form-field
       *ngIf="form.group.controls.id; let ctrl"
       id="form-field-id"
       color="accent">
       <mat-label>ID</mat-label>
-      <input matInput autocomplete="off" [formControl]="ctrl" />
+      <input matInput autocomplete="newvalue" [formControl]="ctrl" />
```

Comment:
every PR we try another value? 😄

Reply:
Unfortunately, not able to find permanent fix to this recurring issue :(

Reply:
I mean, we tried, can't do more than that. Also can't really fight chrome if chrome really wants to do what it wants